### PR TITLE
fix: minds ui

### LINKED ISSUE #4716: [Bug] Incorrect injection place
### How to reproduce?

截屏2021-10-30 02 13 40

### Mask Version:

- [x] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### LINKED ISSUE #4745: [Bug] Minds minor UI issues
Great job on the Minds.com integration! It looks awesome. Here are some minor UI issues I've encountered:

Text overflows in comments

image

Mask icon appears outside of the composer when the user is not connected to the site.
image

I'll update this issue if I see any other issues

### LINKED ISSUE #4762: [Bug] CORS issue when request resources in minds
### How to reproduce?

image

### Mask Version:

- [x] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by zhouhanseng**
closes #4745
closes #4716
closes #4762

### CODE REVIEW

File: packages/mask/src/extension/background-script/WelcomeService.ts [hunk 1]
```diff
@@ -112,7 +112,7 @@ async function createBackupInfo<T>(obj: T, type?: 'txt' | 'json') {
 export async function openOptionsPage(route?: DashboardRoutes, search?: string) {
     return browser.tabs.create({
         active: true,
-        url: browser.runtime.getURL(`/dashboard.html#/${route}${search ? `?${search}` : ''}`),
```
Comment (zhouhanseng) [inline packages/mask/src/extension/background-script/WelcomeService.ts:115]: dashboard.html#//setup => dashboard.html#/setup

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

File: packages/mask/src/extension/background-script/WelcomeService.ts [hunk 2]
```diff
@@ -112,7 +112,7 @@ async function createBackupInfo<T>(obj: T, type?: 'txt' | 'json') {
...
+        url: browser.runtime.getURL(`/dashboard.html#${route}${search ? `?${search}` : ''}`),
```
Comment (zhouhanseng) [inline packages/mask/src/extension/background-script/WelcomeService.ts:115]: `dashboard.html#//setup` => `dashboard.html#/setup`

**Comment by wenluomask** [comment]
If the decrypted comment is too long, text flow exists and encrypted reply to reply need some UI adjustment;
截屏2021-11-08 下午5 26 04

**Comment by zhouhanseng** [comment]
@wenluomask fixed. And the text overflow here belongs to minds itself.

image

**Comment by wenluomask** [comment]
Will inject twice for comment.
https://user-images.githubusercontent.com/85149605/140870035-65ad332b-ea0c-4692-88aa-902d867337c1.mov

**Comment by zhouhanseng** [comment]
@wenluomask This Inject bug belongs to minds itself. Try usual comment without encryption.

**Comment by wenluomask** [comment]
= =...It is...

Checked for this pr.

**Comment by wenluomask** [comment]
Actually, the text color for Dark mode is different from my last test, and I was trying to change theme to Light mode but it failed. Could you please check what kind of the problem is?

截屏2021-11-11 上午10 52 28

**Comment by zhouhanseng** [comment]
@wenluomask fixed.

**Comment by wenluomask** [comment]
It doesn't work for Light Mode now......
截屏2021-11-12 下午1 37 57
截屏2021-11-12 下午1 38 11
截屏2021-11-12 下午1 38 21

**Comment by zhouhanseng** [comment]
@wenluomask fixed

**Comment by wenluomask** [comment]
Sorry for the delay.
Checked.

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**